### PR TITLE
AI-105: Fix update contributions issues

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
**Before**

continue-btn

update-table-issue

update-btn

**After**

resovled-continue-btn

resolved-update-table-issue

resolved-update-btn

### CODE REVIEW

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss [hunk 1]
```diff
@@ -83,6 +83,121 @@
...
+      margin: 0;
+      padding: 15px 10px 23px;
+    }
+
+    .columnheader {
+      border-bottom: 1px solid $crm-dark-grayblue;
+      border-left: none !important;
+      border-right: none !important;    
+
+      th,td {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Leave a space after , or split it over two lines

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss [hunk 2]
```diff
@@ -83,6 +83,121 @@
...
+      }
+
+      td {
+        border: none;
+        color: $crm-copy;
+        padding: 10px 20px;
+      }
+    }
+
+    input[type="text"],.select2-choice {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Leave a space after, or split it over two lines

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss (hunk 1 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss (hunk 2 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss [hunk 3]
```diff
@@ -83,6 +83,123 @@
...
+        width: 78%;
+      }
+
+      .select2-chosen {
+        color: $gray-darker;
+        font-size: $font-size-base;
+        margin-right: 0;            
+      }
+
+      abbr {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Is it a class name or tag? if class name plz add .

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss [hunk 4]
```diff
@@ -83,6 +83,123 @@
...
+        background: none;
+      }
+
+      .select2-arrow{
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Leave a space before {

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss [hunk 5]
```diff
@@ -133,6 +250,30 @@
...
+          background: $crm-blue-light;
+        }
+        
+        &.cancel {
+          background: $crm-white;
+          border: 1px solid $crm-copy;
+          color: $gray-darker;
+        
+          &:hover {
+          background: $crm-copy;
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Adjust the indentation

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss (hunk 3 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: This is tag not a class.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss (hunk 4 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss (hunk 5 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_contributions.scss]: Fixed.

**Pull request merged**